scraping/spiders/france_red_cross.py

```diff
@@ -0,0 +1,200 @@
+import re
+import logging
+
+import scrapy
+
+
+log = logging.getLogger(__name__)
+
+
+CATEGORIES = {
+    'domicile': 'Accommodation',
+    'enfants & familles': 'Children',
+    'personnes handicapees': 'Disability support',
+    "lutte contre l'exclusion": 'Social Help',
+    'sanitaire': 'Medical',
+}
+
+SERVICES_TO_CATEGORIES = {
+    'aide alimentaire': 'Food',
+    'aides financières': 'Finance',
+    'textile & bric-à-brac': 'Clothes',
+    'accès aux soins et bien être': 'Medical',
+    'accueil et orientation': 'Accommodation',
+    'accueil famille-enfant': 'Children',
+    'espace bébé parents': 'Children',
+    'dynamique jeunesse': 'Children',
+    'point hygiène': 'Medical',
+    'postes de secours': 'Any help',
+
```

Comment:
I suggest adding a simple unit test that will make sure categories stay up to date:

```python
def test_categories_mapping():
    for category in CATEGORIES.values() + SERVICES_TO_CATEGORIES.values():
        assert category in constants.CATEGORIES
```

Reply:
Makes perfect sense, added.

Reply:
Makes perfect sense, added.